### ISSUE TITLE
Org (Detail view) - round 2

### ISSUE THREAD

**Issue by jamlung-ri** [OpenConceptLab/ocl_issues#1799]
Remaining tasks from previous ticket: #1757

- [x] The design shows two statistics that look like a sum of numbers that come from multiple repo summary APIs: "# Novel Concepts", and "# Versions published". I can envision how each of them are calculated, but does this seem technically feasible?
     - See #1857 for this
- [x] "Most published content" which shows the top 3 most common concept classes seems useful, but again, it comes from a sum of repo summaries.
- [x] Figure out the difference between Pinning and Featuring a repository in an Org
- [x] View additional org attributes and created on date in header
- [x] For Sunny: 
     - [x] Make sure Profile icon appears correctly e.g. [CIEL icon](https://app.v3.qa.openconceptlab.org/#/orgs/CIEL/)
     - [x] Make sure there's a Member overflow for 8+(?) members
- [x] When there is no About entry and no bookmarks then the overview section remains empty. How will we fill in empty orgs? Empty org example: 
![image](https://github.com/OpenConceptLab/ocl_issues/assets/55999682/731a80f6-8e71-4bf1-b55b-3b9614b9cced))

**Comment by paulsonder** [comment]
@jamlung-ri 

If there is no **About** section, no **Pinned Repo's** and the Org does not own any **Repo's**, we should show an **Empty State** - I will add a design.

**Comment by paulsonder** [comment]
Hi @jamlung-ri - the Org designs have been updated. Please could you review them to see if they're ready for development?

Find the updated designs [here](https://zpl.io/DpGd5Kr).

**Comment by jamlung-ri** [comment]
Thanks @paulsonder, this is looking great and seems nearly done. I do have a couple of points that might not need a design, but just want to make sure it's covered. Please check the boxes below if you feel like this is covered either in the designs or if you and Sunny have a shared understanding of this:

- [ ] The Twitter (social media?) icon at the top is one that we either need to figure out how to add into the Create/Edit Organization form, or we can omit it for now. I'm okay either way but lean toward omitting it. This might be something we bring up to Sunny to see if he has thoughts.
- [ ] The truly empty state (i.e. when an org is first created and the user filled out the bare minimum information) may or may not include the About section or Pinned Repositories, but it should always have at least the following activities:
       1. [Organization Name] was created on OCL!
       2. [User] was added to [Organization Name]

If we have these all addressed, then let's assign this ticket to Sunny and move it forward to Dev!

**Comment by paulsonder** [comment]
Hey @jamlung-ri 

Also added a 'truly' empty state to Zeplin :)

**Comment by snyaggarwal** [comment]
@paulsonder Couple of things on the designs:
1. How will custom attributes link work, a popup window?
2. We are not showing Organization's ID (shor-code) anywhere? It will be there in URL only. Is that something intentional?

**Comment by paulsonder** [comment]
Hi @snyaggarwal. Thanks for the feedback.

1. That's right, a modal was the intention here.
2. Good catch, I will update the design.
3. I will create an avatar for Orgs that have not uploaded one.
4. That's correct.

**Design to do:**

- [ ] Add short code
- [ ] Default Org avatar

**Comment by snyaggarwal** [comment]
@paulsonder View all attributes would open a modal with all attributes including custom attributes? Like in a table of two columns? Any design reference? 
For example, check out [PEPFAR:DATIM-MOH-DAA-FY23](https://app.qa.openconceptlab.org/#/orgs/PEPFAR/sources/DATIM-MOH-DAA-FY23/) header, it has popup for custom attributes, which has formatted and Raw options

**Comment by snyaggarwal** [comment]
@paulsonder Whats the CTA to reach to this [Browse members](https://app.zeplin.io/project/623204ca71595ab4a7e3f67e/screen/668514841ad8f42d5b476269)

**Comment by snyaggarwal** [comment]
@paulsonder For now I have added Browse Members Dialog on click of members in Org Statistics

**Comment by snyaggarwal** [comment]
@jamlung-ri @paulsonder Can you guys check Org view and provide feedbacks

**Comment by jamlung-ri** [comment]
@snyaggarwal Overall this is working great! Some quick feedback for you:
* Should there be a "View all attributes" button? I don't see it, and it might be the only place where I can view the org's description and company.

I think we're still waiting on @paulsonder for the short code addition into the design. That and other comments are in the [design](https://zpl.io/RmMWm9z)!

We should note that there are some things that may not be implementable until their dependency is done:
- "Recent activity" feed in the Org overview (waiting on #1885)
- # of followers for the organization (waiting on #1885?)
- "Add a member" button (@paulsonder there is no design for this yet, right? If not, this can wait until we start a later persona)

**Comment by snyaggarwal** [comment]
@jamlung-ri @paulsonder 
We need some design for "View All attributes" as it will have core attributes and/or extras as well. Or should I just use two column table view in a popup for this?

**Comment by jamlung-ri** [comment]
See this comment for user stories and requirements for "View canonical registry" for the org (which should be very similar to what is used for the User Profile)

https://github.com/OpenConceptLab/ocl_issues/issues/1888#issuecomment-2230868277

**Comment by paulsonder** [comment]
> @jamlung-ri @paulsonder We need some design for "View All attributes" as it will have core attributes and/or extras as well. Or should I just use two column table view in a popup for this?

That approach works @snyaggarwal - let me know when you've built it and I'll do a quick review.

**Comment by paulsonder** [comment]
> See this comment for user stories and requirements for "View canonical registry" for the org (which should be very similar to what is used for the User Profile)
> 
> [#1888 (comment)](https://github.com/OpenConceptLab/ocl_issues/issues/1888#issuecomment-2230868277)

I posted a design solution to this in the referenced ticket.

**Comment by snyaggarwal** [comment]
@paulsonder @jamlung-ri Added view all attributes

**Comment by jamlung-ri** [comment]
Awesome, this is looking great! I think we have these outstanding developments before this is done:

- [ ] Add "View Canonical URL Registry" like in [this design](https://zpl.io/n6OQGeY)
- [ ] Activity feed (dependent on #1885) 
- [ ] The following buttons should activate a CTA:
      - [ ] View Canonical URL Registry
      - [ ] Bookmark
      - [ ] Share
- [ ] Discussion: Remove download button and put it into the Share button? This is something that @paynejd and I talked about but didn't solidify.

**Comment by paynejd** [comment]
Quick reactions to Org header based on what has been implemented (not based on designs):
- Org header must include the org ID and "Created" date without going to additional attributes
- Additional attributes implies that you would not show the attributes from the header again, but we are showing everything twice -- can we remove the redundancy?
- Add "X" button to the top-right as an additional way to close the Additional attributes window?
- The additional attributes window should have a title that isn't the name of the org -- why not just "Organization Attributes"? 

Regarding download/share -- in my opinion, we need to define the Download and Share CTAs for each resource and then we can decide if it's a merge or not. I'm not clear without compiling this across all resources.

**Comment by snyaggarwal** [comment]
Added following:
1. "View Canonical URL Registry" link
2. Removed Download button for now
3. ID in Org header
4. Not repeating attributes in popup
5. Additional attributes window's title -- "Organization Attributes".

Need design input on:
1. Add "X" button to the top-right as an additional way to close the Additional attributes window? -- @paulsonder Should we do this for all popups? Its not in MUI5 theme.
2. "Created" date in org header. @paulsonder We have created date, created by, last updated date and last updated by. Any recommendation on how to show this? Or we can just use the same from v2?

**Comment by paulsonder** [comment]
Hi @snyaggarwal 

1. I have created a separate ticket to define requirements to modals and document them more fully as a DS component. I will include the suggestion to include a 'Close' icon. The ticket is #1904 .
2. I will add this meta information to the About panel - I have tagged your for a review in a Zeplin screen.

**Comment by paulsonder** [comment]
@snyaggarwal Thanks for the approval... all Org designs have been updated.

**Comment by snyaggarwal** [comment]
@jamlung-ri @paulsonder Can we close this one?

**Comment by paulsonder** [comment]
@snyaggarwal Could you update the 'Canonical URL' info as per [design](https://zpl.io/JQEZrOr)? The icon in QA is also incorrect.

**Comment by snyaggarwal** [comment]
1. We dont have a summary API for an Org and so we don't have a way to know how many canonical URL entries are there. Happy to change just the design, but now we wont be able to show the count until we update the API.
2. Where can I get the icon from design?

**Comment by paulsonder** [comment]
@snyaggarwal Icon is in [here](https://zpl.io/JQEZrOr).

**Comment by snyaggarwal** [comment]
@paulsonder Added this
